vulkaninfo/generated/vulkaninfo.hpp

```diff
@@ -773,6 +773,30 @@ void DumpVkDeviceGroupPresentModeFlagBitsKHR(Printer &p, std::string name, VkDev
     DumpVkDeviceGroupPresentModeFlagsKHR(p, name, value, width);
     p.ObjectEnd();
 }
+void DumpVkToolPurposeFlagsEXT(Printer &p, std::string name, VkToolPurposeFlagBitsEXT value, int width = 0) {
+    if (value == 0) p.PrintElement("None");
+    if (1 & value) p.SetAsType().PrintElement("TOOL_PURPOSE_VALIDATION_BIT_EXT");
```

Comment:
Any reason for not using the Vulkan enum bit definitions here (i.e., VK_TOOL_PURPOSE_VALIDATION_BIT_EXT) instead of 1, 2,4,  etc.?

Reply:
This code is autogenerated, so there is little reason to change it to the textual version. If there is any changes to the spec, which alters the value, the autogen script will pick up the change and use the correct values.